public/views/clientList.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Position List</title>
+    <link rel="stylesheet" href="../styles/reset.css">
+    <link rel="stylesheet" href="../styles/styles.css">
+    <link rel="stylesheet" href="../styles/modal.css">
+</head>
+<body>
+    <header>
+        <a id="mindSetLogo" href="/"><span>Mind</span>SET</a>
+        <nav>
+          <a href="#">admins</a>
+          <a href="#">applications</a>
+          <a id="clientsNav" href="../views/clientList.html">clients</a>
+          <a href="#">interviews</a>
+          <a href="#">positions</a>
+          <a href="#">profiles</a>
+          <a href="#">psychologists</a>
+          <a href="#">sessions</a>
+        </nav>
+      </header>
+      <div class="container">
+        <h2 class="header">Clients</h2>
+        <table class="list">
+          <thead>
+          <tr>
+            <th>Name</th>
+            <th>Phone</th>
+            <th>Actions</th>
+          </tr>
+        </thead>
+        <tbody id="clients-table-content">
+        </tbody>
+        </table>
+        <button class="button" id="addClient" type="button">
+          Add Client
+        </button>
+      </div>
+      <section id="modal-section" class="modal-display-off">
+        <div id="modal-content">
+            <span id="modal-close-button">&times;</span>
+            <h1 id="modal-title"></h1>
+            <div id="modal-data-inputs"></div>
+            <div id="modal-buttons">
+              <button id="cancel-button">CANCEL</button>
+              <button id="procced-button">CONFIRM</button>
+            </div>
+        </div>
+      </section>
+  <footer>
+    <div>© Quito's group, Radium Rocket "Become a Software Developer 2021"</div>
+  </footer>
+  <script type="text/javascript" src="../js/modal.js"></script>
+    <script type="text/javascript" src="../js/clientList.js"></script>
```

Comment:
put the `<script>` into the `<head>` tag at the top and add `async` attribute

Reply:
Done!